pkg/detectors/airtableapikey/airtableapikey_test.go

```diff
@@ -29,12 +29,15 @@ func TestAirTableApiKey_Pattern(t *testing.T) {
 		{
 			name:  "valid pattern - with key",
 			input: fmt.Sprintf("airtable secrets: %s", validPattern),
-			want:  []string{"key_Yuztyq-pOcv67app_pOcv67-Yuztyq"},
+			want:  []string{"app_pOcv67-Yuztyq:key_Yuztyq-pOcv67"},
 		},
 		{
-			name:  "valid pattern - with personal key",
-			input: fmt.Sprintf("airtable secrets: %s", validPersonalKeyPattern),
```

Comment:
![Screenshot from 2024-11-12 11-03-09](https://github.com/user-attachments/assets/78d39f8e-0bea-41db-96c1-de566d6a1570)
@rgmz linter is failing can you take a look.